packages/xrpl/test/integration/transactions/accountDelete.test.ts

```diff
@@ -1,33 +1,74 @@
-import _ from 'lodash'
+import { assert } from 'chai'
 import { AccountDelete } from 'xrpl-local/models/transactions'
 
 import serverUrl from '../serverUrl'
-import { setupClient, teardownClient } from '../setup'
-import { generateFundedWallet, ledgerAccept, testTransaction } from '../utils'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { generateFundedWallet, runCommand } from '../utils'
 
 // how long before each test case times out
 const TIMEOUT = 20000
 
 describe('AccountDelete', function () {
-  this.timeout(TIMEOUT)
+  let testContext: XrplIntegrationTestContext
 
-  beforeEach(_.partial(setupClient, serverUrl))
-  afterEach(teardownClient)
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
 
-  it('base', async function () {
-    const wallet2 = await generateFundedWallet(this.client)
-    // to the satisfy the condition that account sequence and current ledger_index should be 256 apart.
-    const promises: Array<Promise<void>> = []
-    for (let iter = 0; iter < 256; iter += 1) {
-      promises.push(ledgerAccept(this.client))
-    }
+  it(
+    'base',
+    async () => {
+      const wallet2 = await generateFundedWallet(testContext.client)
+      // to the satisfy the condition that account sequence and current ledger_index should be 256 apart.
+      // const promises: Array<Promise<void> | Promise<unknown>> = []
+      // for (let iter = 0; iter < 256; iter += 1) {
+      //   promises.push(ledgerAccept(testContext.client))
+      // }
+      // await Promise.all(promises)
```

Comment:
Should this code be deleted?

Reply:
@JST5000 I don't think so, because we should re-add it at some later point WRT to the comment I left in the file

Reply:
Can we leave a TODO then?

Reply:
Done